tests/inspector/ckeditorinspector.js

```diff
@@ -3,18 +3,19 @@
  * For licensing, see LICENSE.md.
  */
 
-/* global document, window */
+/* global document, window, HTMLElement */
 
 import TestEditor from '../utils/testeditor';
 import CKEditorInspector from '../../src/ckeditorinspector';
 import Logger from '../../src/logger';
 
 describe( 'CKEditorInspector', () => {
-	let editor, element, inspectorRef;
+	let editor, element, inspectorRef, warnStub;
```

Comment:
I'd move the `inspectorRef` to the test scopes as it doesn't need to be used anywhere else.

Reply:
It's used nearly everywhere. I wouldn't bother.